### ISSUE TITLE
Cannot start release 1.1 in Windows 10 Pro 10.0.19045

### ISSUE THREAD

**Issue by Supermanuu** [hypertensiune/Mousedroid#3]
Hi!

I cannot start the Mousedroid server in Windows 10. I have downloaded the release and unzipped it on my filesystem. I have started it from this batch script that I have created in the unzipped folder:

```batch
setlocal
SET PATH=%PATH%;C:\MinGW\bin

%~dp0/Mousedroid.exe
```

This is the setup:
![image](https://github.com/hypertensiune/Mousedroid/assets/17361561/0cc7540c-dae1-4637-986d-a8593a343eca)

The restult was this popup:
![image](https://github.com/hypertensiune/Mousedroid/assets/17361561/0f79dc89-c347-445e-ba1f-905385bf86e0)
Traduced to english this meams that `libgcc_s_seh-1.dll` is missing.

I have been looking for `libgcc_s_seh-1.dll` in the MinGW Installation Manager but I could not find it.

Could you help me?

**Comment by Supermanuu** [comment]
Ok, I found the problem!

It seems that you use MinGW-w64 to build this! I have downloaded it from https://github.com/niXman/mingw-builds-binaries and now works fine!

This is the updated script:

`
setlocal
SET PATH=%PATH%;C:\MinGW-w64\mingw64\bin

%~dp0/Mousedroid.exe
`

I installed MinGW in `C:\MinGW-w64`, of course.

I suggest you to add this into the readme file to help other Windows users to start the program.
Thank you.
